### DISCUSSION TITLE
Color plugin needed manual load

### DISCUSSION THREAD

**Discussion by Hackmodford** [MvvmCross/MvvmCross#4127] [Q&A]
I had to add this to my iOS setup class in order to convert to native colors.

```C#
public override void LoadPlugins(IMvxPluginManager pluginManager)
{
    base.LoadPlugins(pluginManager);
    pluginManager.EnsurePluginLoaded<MvvmCross.Plugin.Color.Platforms.Ios.Plugin>();
}
```

Is this expected behavior? If so I will add it to the doc.

**Comment by Cheesebaron** [comment]
@Hackmodford is this only an issue on iOS? `LoadPlugins` _should_™ handle this already, but there might be an issue on iOS.

**Reply by Hackmodford** [reply]
Didn't get a chance to try it in Android. This particular app was an iOS only prototype.
